stable/kommander/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
Hmmm, i honestly forget if the namespace is created or automagically for you.

Reply:
It should get all created at the same time.